### PR TITLE
Hardening/1136 Add Table of Contents

### PR DESCRIPTION
**Pull request by raconte**
Added a table of contents, for those files:
-database_model.md,database_admin.md,logs.md,filtering.md and diagnosis.md.

### CODE REVIEW

File: doc/manuals/admin/database_admin.md [hunk 1]
```diff
@@ -19,9 +34,11 @@ mongodump --host <dbhost> --db <db>
 This will create the backup in the dump/ directory.
 
 Note that if you are using
-[multitenant/multiservice](#multitenant/multiservice-database-separation)
+[multitenant/multiservice](#multiservicemultitenant-database-separation)
```
Comment (fgalan) [inline doc/manuals/admin/database_admin.md]: Is this correct? The "/" has been removed...

File: doc/manuals/admin/database_admin.md [hunk 2]
```diff
@@ -39,9 +56,11 @@ mongorestore --host <dbhost> --db <db> dump/<db>
 ```
 
 Note that if you are using
-[multitenant/multiservice](#multitenant/multiservice-database-separation)
+[multitenant/multiservice](#multiservicemultitenant-database-separation)
```
Comment (fgalan) [inline doc/manuals/admin/database_admin.md]: Is this correct? The "/" has been removed...

File: doc/manuals/admin/database_admin.md [hunk 3]
```diff
@@ -63,12 +82,14 @@ to take into account:
...
         authorizing on `admin` DB ensures permisions in all them.
+     
+[Top](#top)
 
 ## Multiservice/multitenant database separation
 
 Normally, Orion Context Broker uses just one database at MongoDB level
 (the one specified with the `-db` command line option, typically "orion").
-However, when [multitenant/multiservice](#multitenant/multiservice-database-separation) is used
+However, when [multitenant/multiservice](#multiservicemultitenant-database-separation) is used
```
Comment (fgalan) [inline doc/manuals/admin/database_admin.md]: Is this correct? The "/" has been removed...

File: doc/manuals/admin/upgrading_crossing_0-14-1.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ upgrading procedure.
     [swap\_coords.js](https://github.com/telefonicaid/fiware-orion/tree/0.14.1-FIWARE-3.5.1/scripts/managedb/swap_coords.js)
 -   Apply the script to your DBs, using the following (where 'db' is the
     database name). Note that if you are using
-    [multitenant/multiservice](database_admin.md#multiservice/multitenant-database-separation)
+    [multitenant/multiservice](database_admin.md#multiservicemultitenant-database-separation)
```
Comment (fgalan) [inline doc/manuals/admin/upgrading_crossing_0-14-1.md]: Is this correct? The "/" has been removed...

File: doc/manuals/admin/upgrading_crossing_0-19-0.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ at least one entity document.
 
 -   Apply the script to your DBs, using the following (where 'db' is the
     database name). Note that if you are using
-    [multitenant/multiservice](database_admin.md#multiservice/multitenant-database-separation)
+    [multitenant/multiservice](database_admin.md#multiservicemultitenant-database-separation)
```
Comment (fgalan) [inline doc/manuals/admin/upgrading_crossing_0-19-0.md]: Is this correct? The "/" has been removed...

File: doc/manuals/admin/upgrading_crossing_0-21-0.md [hunk 6]
```diff
@@ -23,7 +23,7 @@ at least one entity document.
 
 -   Apply the fix\_location\_gjson.py to your DBs, using the following
     (where 'db' is the database name). Note that if you are using
-    [multitenant/multiservice](database_admin.md#multiservice/multitenant-database-separation)
+    [multitenant/multiservice](database_admin.md#multiservicemultitenant-database-separation)
```
Comment (fgalan) [inline doc/manuals/admin/upgrading_crossing_0-21-0.md]: Is this correct? The "/" has been removed...

File: doc/manuals/admin/upgrading_crossing_0-21-0.md [hunk 7]
```diff
@@ -39,7 +39,7 @@ at least one entity document.
 -   Apply the attrsvector2attrsobject.py to your DBs, using the
     following (where 'db' is the database name). Note that if you are
     using
-    [multitenant/multiservice](database_admin.md#multiservice/multitenant-database-separation)
+    [multitenant/multiservice](database_admin.md#multiservicemultitenant-database-separation)
```
Comment (fgalan) [inline doc/manuals/admin/upgrading_crossing_0-21-0.md]: Is this correct? The "/" has been removed...

File: doc/manuals/user/multitenancy.md [hunk 8]
```diff
@@ -16,7 +16,7 @@ to other services/tentants. For example, queryContext on tenantA space
 will never return entities/attributes from tenantB space. This isolation
 is based on database separation, which [details are described in the
 Installation and Administration
-manual](../admin/database_admin.md#multiservice/multitenant-database-separation).
+manual](../admin/database_admin.md#multiservicemultitenant-database-separation).
```
Comment (fgalan) [inline doc/manuals/user/multitenancy.md]: Is this correct? The "/" has been removed...

File: doc/manuals/admin/database_admin.md (hunk 3 above)
Comment (raconte) [inline doc/manuals/admin/database_admin.md]: Needed to make the link work. NTC

File: doc/manuals/admin/database_admin.md (hunk 2 above)
Comment (raconte) [inline doc/manuals/admin/database_admin.md]: Needed to make the link work. NTC

File: doc/manuals/admin/database_admin.md (hunk 1 above)
Comment (raconte) [inline doc/manuals/admin/database_admin.md]: Needed to make the link work. NTC

File: doc/manuals/admin/upgrading_crossing_0-14-1.md (hunk 4 above)
Comment (raconte) [inline doc/manuals/admin/upgrading_crossing_0-14-1.md]: Needed to make the link work. NTC

File: doc/manuals/user/multitenancy.md (hunk 8 above)
Comment (raconte) [inline doc/manuals/user/multitenancy.md]: Needed to make the link work. NTC

File: doc/manuals/admin/upgrading_crossing_0-21-0.md (hunk 7 above)
Comment (raconte) [inline doc/manuals/admin/upgrading_crossing_0-21-0.md]: Needed to make the link work. NTC

File: doc/manuals/admin/upgrading_crossing_0-21-0.md (hunk 6 above)
Comment (raconte) [inline doc/manuals/admin/upgrading_crossing_0-21-0.md]: Needed to make the link work. NTC

File: doc/manuals/admin/upgrading_crossing_0-19-0.md (hunk 5 above)
Comment (raconte) [inline doc/manuals/admin/upgrading_crossing_0-19-0.md]: Needed to make the link work. NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**